docs/src/templates/pages/styleguide/ui-templates/page-components/cookie-banner.hbs

```diff
@@ -0,0 +1,361 @@
+---
+title: Component Library
+docs: false
+
+layout: site-frame
+styleguide: true
+---
+
+<div id="CookieAnchor"></div>
```

Comment:
Looks like the `l-container` doesn't need to be indented below?

Reply:
We should probably be binding using an attribute here rather than an id.